pyfrc/test_support/controller.py

```diff
@@ -78,6 +78,22 @@ def run_robot(self):
             self._robot_finished = True
             robot.endCompetition()
 
+            if isinstance(self._reraise.exception, RuntimeError):
+                if str(self._reraise.exception).startswith(
+                    "HAL: A handle parameter was passed incorrectly"
+                ):
+                    msg = (
+                        "Do not reuse HAL objects in tests! This error may occur if you"
+                        + " stored a motor/sensor as a global or as a class variable"
+                        + " outside of a method."
```

Comment:
nit: prefer implicit string literal concatenation?
```suggestion
                        " stored a motor/sensor as a global or as a class variable"
                        " outside of a method."
```